schema.sql

```diff
@@ -6,7 +6,9 @@ create table users (
   -- UUID from auth.users
   id uuid references auth.users not null primary key,
   first_name text,
-  last_name text
+  last_name text,
+  -- The URL for the user's avatar. Images should be stored outside of the database.
+  avatar_url text default "/avatar.png",
```

Comment:
As discussed, can be done with blobs or [hashes](https://github.com/woltapp/blurhash), but simple is good

Reply:
If we want to simplify this, we can use https://unavatar.now.sh/

Reply:
that's very cool. @thorwebdev - will leave that decision to you - we can remove the field if we use unavatar (will everyone have email though?). 

Default to https://unavatar.now.sh/facebook/zuck so they are incentivised to update their profile

Reply:
Happy to use https://unavatar.now.sh/. @leerob would we put that straight into the source code? Then we can remove the avatar_url column here, correct?

Reply:
If we can't figure out their avatar based on their email, then we should use the gradient circle like here -> https://demo.vercel.store/

Reply:
@leerob okay, I've removed it from the schema. Can I leave the avatar to you?

Reply:
Yep, I can do that. When's your deadline again?

Reply:
End of this week would be stellar. Then I can record the workshop over the weekend. Do you think it's feasible to have the project moved to the Vercel GitHub org by then?

Reply:
Yeah I think so.